src/main/scala/org/constellation/infrastructure/endpoints/TransactionEndpoints.scala

```diff
@@ -68,12 +68,19 @@ class TransactionEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F]
 
   def peerEndpoints(transactionService: TransactionService[F], metrics: Metrics) =
     getTransactionEndpoint(transactionService) <+>
-      getBatchEndpoint(metrics, transactionService)
+      getBatchEndpoint(metrics, transactionService) <+>
+      getLastTransactionEndpoint(transactionService)
 
   private def getTransactionEndpoint(transactionService: TransactionService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
     case GET -> Root / "transaction" / hash =>
       transactionService.lookup(hash).map(_.asJson).flatMap(Ok(_))
   }
+
+  /** Gets the reference to the last transaction for a given address or an empty reference if no transactions yet. */
+  private def getLastTransactionEndpoint(transactionService: TransactionService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "last-transaction" / address =>
```

Comment:
first of all, it should be `/transaction/last` instead of `last-transaction`. Secondly, it should be `ref` not `tx` :)
I'd say, the endpoint can be `/transaction/last-ref?sender={address}`. What do you think?

Reply:
Right, ref! As for the naming I was under an impression that we embbed params in the url instead of using query params, hence I used this :). I think the endpoint name proposed by you sound good, will change.

Reply:
@marcinwadon It's actually now conflicting with getTransactionEndpoint:
/transaction/{hash}

What would you say to:
/{address}/transaction/last-ref e.g. /DAG123/transaction/last-ref

Reply:
Or the other way round with address at the end:
/transaction/last_ref/{address}

Reply:
definitely not `{address}/..` because it can conflict with other endpoints as the wildcard will be at the top level.
re: `/transaction/last-ref?sender={address}`, please check if proper order of those endpoints will work. (firstly all explicit, then wildcards)
if no, what about: `/transaction-ref?sender={address}`?

Reply:
I would actually go with path variable instead of query param as we are identifying a resource here, according to:
https://medium.com/@fullsour/when-should-you-use-path-variable-and-query-parameter-a346790e8a6d

the question is:
/transaction/last-ref/{address}
vs
/transaction-ref/last/{address}

I would say the first one sounds better/is more descriptive for me :).
WDYT @kpudlik @marcinwadon 
I guess we may be creating a last-accepted endpoint in the future also - so we should keep that in mind...

Reply:
I'd opt for `/transaction/last-ref?sender{address}` later in the future as it imo suits perfectly for that use case

Reply:
because right now you're not able to distinguish between `sender` and `receiver` which we would need